main.py

```diff
@@ -1,5 +1,6 @@
 import utils
 from cvesCollection import CVEsCollection
+from cve import Cve
 
```

Comment:
utils: this file isn't really utils. create a class that downloads the db and allows retrieving the downloaded db (per year, like the util function)

Reply:
done